AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Campaign/Edit.cshtml

```diff
@@ -73,7 +73,7 @@
                     <input type="hidden" asp-for="ImageUrl" id="image-url-input" />
                 }
                 <div class="form-group">
-                    <label asp-for="Name" class="col-md-2 control-label"></label>
+                    <label asp-for="Name" class="col-md-2 control-label">Campaign Title</label>

```

Comment:
The label actually binds from the property in the ViewModel and we can actually achieve this by adding a DisplayName attribute to the property in question. See #2252

Could this be updated to that approach please?